docs_src/pipelinerun.md

```diff
@@ -1,128 +1,78 @@
-
-!!!Note there could be multiple containers with IMAGE "openvino/ubuntu20_data_runtime:2021.4.2", depending on number of sample-media video file you have.
-
-!!! failure
-    If you do not see all of the above docker containers, look through the console output for errors. Sometimes dependencies fail to resolve and must be run again. Address obvious issues. To try again, repeat [Run camera simulator](#run-camera-simulator).
-
----
-## Run pipeline with different input source(inputsrc) types
-Use docker-run.sh to run the pipeline, here is the table of basic scripts for each combination:
-
-| Input source Type | scripts                                                                                                |
+| Input Source Type | Scripts                                                                                                |
 | ----------------- | -------------------------------------------------------------------------------------------------------|
-| simulated camera  | <code>sudo ./docker-run.sh --platform core&#124;xeon&#124;dgpu.x --inputsrc rtsp://127.0.0.1:8554/camera_0</code>|
-| realsense camera  | <code>sudo ./docker-run.sh --platform core&#124;xeon&#124;dgpu.x --inputsrc <serial_number> --realsense_enabled</code>        |
+| Simulated camera  | <code>sudo ./docker-run.sh --platform core&#124;xeon&#124;dgpu.x --inputsrc rtsp://127.0.0.1:8554/camera_0</code>|
+| RealSense camera  | <code>sudo ./docker-run.sh --platform core&#124;xeon&#124;dgpu.x --inputsrc <serial_number> --realsense_enabled</code>        |
 | USB camera        | <code>sudo ./docker-run.sh --platform core&#124;xeon&#124;dgpu.x --inputsrc /dev/video0</code>                         |
-| a video file      | <code>sudo ./docker-run.sh --platform core&#124;xeon&#124;dgpu.x --inputsrc file:my_video_file.mp4</code>             |
+| Video file      | <code>sudo ./docker-run.sh --platform core&#124;xeon&#124;dgpu.x --inputsrc file:my_video_file.mp4</code>             |
+    
 
-!!!Note for `dgpu.x`, the x can be 0, 1, 2, ... depends on how many discrete gpu it has in the system
-
----
-## Optional parameters
-The optional parameters you can apply to the docker-run.sh input, just note that they will affect the performance of pipeline run.
+**_Note:_**  The value of x in `dgpu.x` can be 0, 1, 2, and so on depending on the number of discrete GPUs in the system.
+    
+### Optional Parameters
 
-### `--classification_disabled`
-To disable classification process of image extraction when applying model, default is NOT disabling classification if this is not provided as input to docker-run.sh
+The following are the optional parameters that you can provide as input to `docker-run.sh`. Note that these parameters would affect the performance of the pipeline. 
+    
+- `--classification_disabled`: Disables the classification process of image extraction when applying model. By default, the classification is enabled. 
```

Comment:
What does "when applying model" mean? It is not clear.

Reply:
when taking the model IR files into the process to analyze the input source

Reply:
I am still did not understand the explanation. I have tried to rephrase:

Disables the classification process of image extraction when the model IR files are analyzed for the input source. By default, the classification is enabled. 


Is the second part of the sentence necessary. Can it be written as:

Disables the classification process of image extraction. By default, the classification is enabled.

Reply:
I think the single sentence is clear enough.

Disables the classification process of image extraction. By default, the classification is enabled.